webapi/src/main/scala/org/knora/webapi/routing/v1/SearchRouteV1.scala

```diff
@@ -74,6 +76,9 @@ object SearchRouteV1 extends Authenticator {
             case other => None
         }
 
+        //
+        // the params with multiple values have angle brackets appended: param name[]
+        //
```

Comment:
You are right, the square brackets are not needed any longer (jQuery traditional mode: https://github.com/dhlab-basel/Knora/commit/1a37c9c85abe686fe277b46a5c572b9b3641a08c).